src/Lean/MetavarContext.lean

```diff
@@ -252,7 +256,13 @@ structure MetavarDecl where
   userName       : Name := Name.anonymous
   lctx           : LocalContext
   type           : Expr
-  depth          : Nat
+  depth          : Nat /--
```

Comment:
This docstring must be before `depth : Nat`

Reply:
Done!